core/controllers/classifier.py

```diff
@@ -123,3 +129,61 @@ def post(self):
         classifier_services.mark_training_job_complete(job_id)
 
         return self.render_json({})
+
+
+def update_failed_jobs(job_models):
+    for job_model in job_models:
+        classifier_services.mark_training_job_failed(job_model.id)
+
+
+def fetch_training_data(exp_id, state):
+    exp_model = exp_services.get_exploration_by_id(exp_id)
+    state_model = exp_model.states[state]
+    training_data = state_model.get_training_data()
+    return training_data
+
+
+def fetch_next_job():
+    classifier_job_models = (
+        classifier_services.get_all_classifier_training_jobs())
```

Comment:
@prasanna08 I have updated it slightly now. Do you still suggest storing cursor as an attribute or in memcacahe and using it to paginate the query results?